salt/states/zabbix_host.py

```diff
@@ -291,6 +299,13 @@ def _interface_format(interfaces_data):
         if host_updated_params:
             update_host = True
 
+        if "inventory_mode" in host_extra_properties:
+            inventory_mode = host_extra_properties["inventory_mode"]
+        elif host["inventory_mode"] == "-1":
+            inventory_mode = "0"
+        else:
+            inventory_mode = host["inventory_mode"]
+
```

Comment:
**question** would this be clearer written like:

```
inventory_mode = host_extra_properties.get("inventory_mode", host["inventory_mode"])
inventory_mode = "0" if inventory_mode == "-1" else inventory_mode
```
?

Reply:
(looking at the below code - it might be better as:

```
host_mode = host["inventory_mode"]
inventory_mode = host_extra_properties.get("inventory_mode", "0" if host_mode == "-1" else host_mode)
```

Reply:
Looks good and better to understand. Any problem if I eliminate the `host_mode` and just use `host["inventory_mode"]` in the `inventory_mode` line, or change it to `host_inventory_mode`? `host_mode` looks easy to be mistaken as a variable related to `host` status and not limited to inventory scope.

Reply:
Made the change, but instead `host_mode` I did use `host_inventory_mode`